### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by MICHAELKITH**
### In this milestone 
I have added index.html and style.css files in the project

### CODE REVIEW

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @MICHAELKITH 
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- .gitignore file created ✔️ 
- Linters are green ✔️


 ## Required Changes ♻️

- [ ] **_🛑  to make your readme more professional, kindly update it to have all of the sections shown in this [template ](https://github.com/microverseinc/readme-template/blob/master/README.md) & Please make sure to fill all of the fields with information about your project 🙏_**



## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MICHAELKITH** [comment]
@ichala 
okay , thank you for the guidance

File: index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 class="header">
+        Hello Microverse!
+    </h1>
+
```
Comment (BregornOriginal) [inline index.html:14]: - [ ] Kindly changes your readme as the reviewer said in the previous review. Remember that you need to make these changes on your actual branch, not on the main.

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @MICHAELKITH   👏🏼 👏🏼 👏🏼 

## Good job so far! 🥳 🥳 🥳 

## There are some issues that you still need to work on to go to the next project but you are almost there!

one-step-closer-one-more

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Excelent Pull Request description 💯 💯 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (MICHAELKITH) [reply index.html:14]: @BregornOriginal i think adding my readme file in main is okay because at the end  I will merge

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @MICHAELKITH 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Setup a "Hello Microverse" project

## Optional suggestions

- [x] Kindly I recommend you to list out all headers of READEME file as suggested by the previous reviewer..

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MICHAELKITH** [comment]
@Abenezer-Tilahun thank you bro, you are professional!

**Pull request merged**